velox/common/memory/MemoryAllocator.h

```diff
@@ -389,6 +389,10 @@ class MemoryAllocator : public std::enable_shared_from_this<MemoryAllocator> {
   const std::vector<MachinePageCount>
       sizeClassSizes_{1, 2, 4, 8, 16, 32, 64, 128, 256};
 
+  std::atomic<MachinePageCount> numAllocated_{0};
+  // When using mmap/madvise, the current of number pages backed by memory.
+  std::atomic<MachinePageCount> numMapped_{0};
```

Comment:
It feels like details of the derived class MmapAllocator are leaking into the base class. Is this intentional?

Reply:
It is actually from MallocAlloctor but is commented for MmapAllocator. Let me update the comment.